packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -144,19 +144,17 @@
 		</template>
 	</tera-infinite-canvas>
 	<tera-progress-spinner v-else :font-size="2" is-centered />
-	<Teleport to="body">
-		<component
-			v-if="dialogIsOpened && currentActiveNode"
-			:is="registry.getDrilldown(currentActiveNode.operationType)"
-			:node="currentActiveNode"
-			@append-output="(event: any) => appendOutput(currentActiveNode, event)"
-			@update-state="(event: any) => updateWorkflowNodeState(currentActiveNode, event)"
-			@update-status="(event: any) => updateWorkflowNodeStatus(currentActiveNode, event)"
-			@select-output="(event: any) => selectOutput(currentActiveNode, event)"
-			@close="addOperatorToRoute(null)"
-			@update-output-port="(event: any) => updateOutputPort(currentActiveNode, event)"
-		/>
-	</Teleport>
+	<component
+		v-if="dialogIsOpened && currentActiveNode"
+		:is="registry.getDrilldown(currentActiveNode.operationType)"
+		:node="currentActiveNode"
+		@append-output="(event: any) => appendOutput(currentActiveNode, event)"
+		@update-state="(event: any) => updateWorkflowNodeState(currentActiveNode, event)"
+		@update-status="(event: any) => updateWorkflowNodeStatus(currentActiveNode, event)"
+		@select-output="(event: any) => selectOutput(currentActiveNode, event)"
+		@close="addOperatorToRoute(null)"
+		@update-output-port="(event: any) => updateOutputPort(currentActiveNode, event)"
+	/>
```

Comment:
I am not exactly sure what is the ramification here, the drilldowns were originally designed such that they can be opened into a new tab/window. Now that is not entirely true anymore because some operators have dependencies on the node to kick things off or to set a flag, but nevertheless I am wondering if we still want to be able to "open in a new tab" and what this might look like? @YohannParis

Reply:
This change assumes the drilldown will always be a modal, (I wrapped `tera-drilldown` with a `<teleport>`). But if we still want to open the drilldown in a new window then this change gets in the way of that.

Reply:
Yes, we still want to open drilldown on new windows, that would really help the UX, specially once we implement the arrows to move up and down the worflow

Reply:
Ok I'll revert this